File: leetcode/100-150/118-peakIndexInMountainArray.py
def peakIndexInMountainArray(self, A):
    i = 0
    while i < len(A)-1 and A[i] < A[i+1]: i+=1
    return i

def peakIndexInMountainArray2(self, A):
    for i in range(len(A)):
        if (A[i] > A[i+1]):
            return i

File: leetcode/0-250/118-peakIndexInMountainArray.java
    public int peakIndexInMountainArray(int[] A) {
        int i = 0, len = A.length;
        while (i < len-1 && A[i] < A[i+1]) i++;
        return i;
    }

    public int peakIndexInMountainArray2(int[] A) {
        int lo = 0, hi = A.length - 1;
        while (lo < hi) {
            int mi = lo + (hi-lo)/2;
            if (A[mi] < A[mi+1]) lo = mi+1;
            else hi = mi;
        }
        return lo;
    }